tests/examples/helm-api-versions/helm-api-versions/templates/test-pod.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if and ( .Capabilities.APIVersions.Has "foo" ) ( .Capabilities.APIVersions.Has "bar") }}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: api-versions-test
```

Comment:
If you add `namespace: {{ .Release.Namespace }}` here, you can skip the transformation on the Chart class, and use the namespace parameter instead.